scripts/exrrfs_ioda_bufr.sh

```diff
@@ -70,6 +75,27 @@ for yaml in "${yaml_list[@]}"; do
 done
 
 # --------------------------------------------------
+# run  bufr2ioda tool for atms bufr obs
+# --------------------------------------------------
+${cpreq} "${PARMrrfs}/atms_beamwidth.txt" .
+${cpreq} "${PARMrrfs}/bufr_atms_mapping.yaml" .
+input_file="atmsbufr"
+output_file="rap.t${cyc}z.atms_{splits/satId}.tm00.nc"
+yaml="bufr_atms_mapping.yaml"
+if [[ -f "$input_file" ]]; then
+  ./bufr2netcdf.x "$input_file" "$yaml" "$output_file"
+  if [[ -f "rap.t${cyc}z.atms_npp.tm00.nc" ]]; then
+     ${cpreq} "rap.t${cyc}z.atms_npp.tm00.nc" "ioda_atms_npp.nc"
+  fi
+  if [[ -f "rap.t${cyc}z.atms_n20.tm00.nc" ]]; then
+     ${cpreq} "rap.t${cyc}z.atms_n20.tm00.nc" "ioda_atms_n20.nc"
+  fi
+  if [[ -f "rap.t${cyc}z.atms_n21.tm00.nc" ]]; then
+     ${cpreq} "rap.t${cyc}z.atms_n21.tm00.nc" "ioda_atms_n21.nc"
+  fi
+else
+  echo "Input file $input_file does not exist."
+fi
```

Comment:
I don't see `bufr_atms_mapping.yaml` anywhere but I may have just overlooked it. Also, is there a way to just specify the output filename from `bufr_atms_mapping.yaml` to just be `ioda*.nc` instead of outputting as `rap*.nc` then renaming?

Reply:
Thanks @delippi, I did forget to commit the bufr_atms_mapping.yaml. Just push this yaml file into parm directory. I think I can change the rap*.nc in the scripts. I will update it.

Reply:
@delippi I just commit the change of the exrrfs_ioda_bufr.sh with the updated atms ioda output file ioda*.nc